spec/syntax.md

```diff
@@ -91,150 +91,198 @@ The syntax specification takes into account the following design restrictions:
 
 _This section is non-normative._
```

Comment:
If this section is now introducing normative text, this statement should be removed.